packages/material-ui/src/RadioGroup/RadioGroup.js

```diff
@@ -9,6 +9,15 @@ import { createChainedFunction, find } from '../utils/helpers';
 class RadioGroup extends React.Component {
   radios = [];
 
+  state = {
+    value: null,
+  };
+
+  constructor(props) {
+    super(props);
```

Comment:
Nit: For consistency don't call it with props for bundle size savings. We don't need `this.props` in the constructor.